specification/Metadata/README.md

```diff
@@ -21,283 +21,227 @@ Draft
+> **Example:** A property storing GPS coordinates might define a range of `[-180, 180]` degrees for longitude values and `[-90, 90]` degrees for latitude values. If the dataset happens to contain only GPS coordinates in a small region, more specific ranges may be given instead.
 
-### Optional Properties and Default Values
+#### Required Properties and No Data Values
 
-Properties can be marked as optional. Optional properties do not need to have associated property values.
+When associated property values must exist for all entities of a class, a property is considered required (`required`).
 
-A property may specify a default value to be used when property values do not exist. The default value must match the property definition, e.g. if `type` is `UINT8` the default value must be an unsigned integer in the range `[0, 255]`. If the property is normalized, the default value is provided in its original integer form, not the normalized form.
+Properties may optionally specify a No Data value (`noData`, or "sentinel value") to be used when property values do not exist. This value must match the property definition, e.g. if `type` is `UINT8` the `noData` value must be an unsigned integer in the range `[0, 255]`. If the property is normalized, the `noData` value is given in its original integer form, not the normalized form.
 
 Individual components in an array cannot be marked as optional; only the array property itself can be marked as optional.
 
-### Semantics
+For array types, `noData` is an array-typed value indicating that the entire array represents a missing value. For example, `[-1, -1, -1]` might be used as a `noData` value for a `VEC3` property. When an array-typed property is required or includes a `noData` value, this has no effect on the interpretation of individual array elements. When variable-length arrays are required, an empty array is still valid.
 
-As mentioned earlier, a property does not have inherent meaning. However it may be assigned a semantic, an identifier that describes how this property should be interpreted in an external semantic specification.
+> **Example:** In the example below, a "tree" class is defined with `noData` indicating a specific enum value to be interpreted as missing data.
+>
+> | property  | componentType | required | noData    |
+> |:----------|:--------------|:---------|:----------|
+> | height    | "FLOAT32"     | ✓        |           |
+> | species   | "ENUM"        |          | "Unknown" |
+> | leafColor | "STRING"      | ✓        |           |
 
 ## Storage Formats
 
 A schema provides the pattern for creating entities. This section covers the various formats and encodings for storing entity metadata. Additional formats and encoding may be defined outside of this specification.
 
-The two formats are defined:
+* **Table format** - property values are stored in parallel 1D arrays
+* **Raster format** - property values are stored in channels of a 2D grid of pixels
```

Comment:
I think we briefly talked about this, could we word this in a way that generalizes to 3D or N-D rasters?

Reply:
Rephrased: 

* **Raster format** - property values are stored in channels of pixel-based formats (usually images)

... The raster format is for storing fine-grained metadata in pixel-based formats such as images and video.